tests/test_evaluate_mesh_on_grants.py

```diff
@@ -37,11 +65,13 @@ def test_evaluate_mesh_on_grants():
         X = texts
         Y = label_binarizer.transform(tags)
 
-        model = MeshCNN()
+        model = MeshXLinear(min_df=1, max_df=10)
         model.fit(X, Y)
-        model.save(model_path)
+        model.save(tmp_dir)
 
         data = pd.DataFrame.from_records(VAL_DATA)
         data.to_excel(data_path, engine="openpyxl")
 
-        evaluate_mesh_on_grants("mesh-cnn", data_path, model_path, label_binarizer_path)
+        evaluate_mesh_on_grants(
+            "mesh-xlinear", data_path, tmp_dir, label_binarizer_path
+        )
```

Comment:
Is this test just checking that you can run this without errors? Or does evaluate_mesh_on_grants produce any outputs that can be checked as well?

Reply:
yeah good point. I think it is just checking everything runs without errors. The output of `evaluate_mesh_on_grants.py` is an F1 score so might be difficult to check this number is as expected.. I might not worry about it

Reply:
actually I did add in an assert to check the F1 is coming out